### PR TITLE
Update Buildah Tutorial to address BZ1867426

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
A bugzilla was created due to a misunderstanding gleaned from the
Buildah tutorial.  The person running the tutorial followed it exactly,
but the example blew up on RHEL for them.  The reason being is the
`--releasever` option has to use a valid version for the host's OS
when it's working on a scratch container or dnf will fail.

This adds clarification to the document to make that clear.

https://bugzilla.redhat.com/show_bug.cgi?id=1867426

Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

#### What type of PR is this?

> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test 
> /kind feature
> /kind flake
> /kind other

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
@tidydata fyi

File: docs/tutorials/01-intro.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ By echoing `$scratchmnt` we can see the path for the [overlay image](https://wik
...
 
-Now that we have a new empty container we can install or remove software packages or simply copy content into that container. So let's install `bash` and `coreutils` so that we can run bash scripts. This could easily be `nginx` or other packages needed for your container.
+Now that we have a new empty container we can install or remove software packages or simply copy content into that container. So let's install `bash` and `coreutils` so that we can run bash scripts. This could easily be `nginx` or other packages needed for your container.  **NOTE:** the version in the example below (30) relates to a Fedora version which is the Linux platform this example was run on.  If you are using the scratch image, the version you specify must be valid for the host or dnf will throw an error.  I.e. If you were to run this on a RHEL platform, you'd need to specify `--releasever 8.1` or similar instead of `--releasever 30`.  If you want the container to be a particular Linux platform, change `scratch` in the first line of the example to the platform you want, i.e. `# newcontainer=$(buildah from fedora)`, and then you can specify an appropriate version number for that Linux platform.
```
Comment (rhatdan) [inline docs/tutorials/01-intro.md:110]: I would prefer the NOTE to be on a new line.  Could you update the "30" to something more recent, say "33" in all of the examples.

Comment (rhatdan) [reply docs/tutorials/01-intro.md:110]: I also thought you like to avoid using "you" in documentation?

Comment (TomSweeneyRedHat) [reply docs/tutorials/01-intro.md:110]: "you" is verboten in man pages, OK in tutorials and blog posts as long as it's not overused.  Will adjust this on Tuesday, now it's almost time for my favorite little man to arrive for a few hours.

Comment (rhatdan) [reply docs/tutorials/01-intro.md:110]: Can you move the NOTE to it's own line.?

Comment (TomSweeneyRedHat) [reply docs/tutorials/01-intro.md:110]: Done, just unburied the inbox, update inc.

Comment (rhatdan) [reply docs/tutorials/01-intro.md]: You also did not update the version from 30 to 33 so that it is a supported version of Fedora for the next 1.5 years.

**Comment by TomSweeneyRedHat** [comment]
@rhatdan The last time I checked F33 didn't work in that line, but it seems to be working now.  Will tweak and thanks for the eagle eye.

**Comment by rhatdan** [comment]
bors retry

**Comment by TomSweeneyRedHat** [comment]
bors retry

**Comment by TomSweeneyRedHat** [comment]
bors retry

**Comment by rhatdan** [comment]
Saving the planet.

**Pull request merged**